dmsrc/sound_len.dm

```diff
@@ -0,0 +1,34 @@
+// Provided a static RSC file path or a raw text file path, returns the duration of the file in deciseconds as a float.
+/proc/rustg_sound_length(file_path)
```

Comment:
can you add support for frequency overrides?

Reply:
No.